harness/determined/pytorch/deepspeed/dsat/_defaults.py

```diff
@@ -43,36 +44,11 @@
     },
 }
 
-AUTOTUNING_DICT = {
-    "enabled": True,
-    "results_dir": "autotuning_results",
-    "exps_dir": "autotuning_exps",
-    "overwrite": False,
-    "metric": "throughput",  # TODO: dynamically populate based on searcher fields.
-    "start_profile_step": 3,
-    "end_profile_step": 5,
-    "fast": True,
-    "max_train_batch_size": None,
-    "mp_size": 1,
-    "num_tuning_micro_batch_sizes": 5,
-    "tuner_type": "model_based",
-    "tuner_early_stopping": 25,
-    "tuner_num_trials": 25,
```

Comment:
did this become `--max-trials`?

Reply:
yep, because we have already established `max_trials` in the determined world to mean the same thing